### ISSUE TITLE
Sympy error in Method of lines analysis example

### ISSUE THREAD

**Issue by fruzsinaagocs** [ketch/nodepy#84]
In the example notebook titled [Method of lines analysis](https://github.com/ketch/nodepy/blob/master/examples/Method%20of%20lines%20analysis.ipynb), running the cell

```python 
rkc = rk.RKC2(10)
rk.linearly_stable_step_size(rkc,centered_diff)
```
gives 
```bash 
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-6a581f261783> in <module>
----> 1 rkc = rk.RKC2(10)
      2 rk.linearly_stable_step_size(rkc,centered_diff)

~/all-virtual-envs/nodepy-env/lib/python3.8/site-packages/nodepy/runge_kutta_method.py in RKC2(m, epsilon)
   3504 
   3505     x=sympy.Symbol('x')
-> 3506     Tm=sympy.special.polynomials.chebyshevt_poly(m,x)
   3507 
   3508     w0=one+sympy.Rational(epsilon,m**2)

AttributeError: module 'sympy' has no attribute 'special'
```
I think the relevant function implementing Chebyshev polynomials moved to `sympy.functions.special.polynomials.chebyshevt` ([sympy docs](https://docs.sympy.org/latest/modules/functions/special.html#chebyshev-polynomials)).

This notebook also relies on `scipy`, which is not a listed dependency.

**Comment by ranocha** [comment]
Thanks for the report. The Chebyshev polynomials are already fixed in c4787f17a8fa36e62716b2d294c3617b25455fd4 - there should just be a new tag. 

@ketch: How would you like to handle the dependency issue?

**Comment by ketch** [comment]
Scipy is now listed as an optional dependency (fixed in 5ff088df09a77f13bdac1ff49bd846bff62447ca).
